src/v2/Apps/Auctions/__tests__/AuctionsApp.jest.tsx

```diff
@@ -83,4 +99,21 @@ describe("AuctionsApp", () => {
     expect(html).not.toContain("Ended")
     expect(html).not.toContain("In Progress")
   })
+
+  it("does not render MyBids or WorksByFollowedArtists if user logged out", () => {
+    useSystemContext.mockImplementation(() => ({
+      user: null,
```

Comment:
**question [non-blocking]:** passing in the user

I see how you're assembling this test case to have a `null` user, but I'm missing how other test cases have a user?? The `mockImplementation` call in the `beforeEach` above only specifies `on` and `off` - where do the other tests get a user from?

*format via [Conventional Comments](https://conventionalcomments.org/)*